src/courses.json

```diff
@@ -29,8 +29,9 @@
   "Einführung in die Informatik ": "INFO",
   "Praktikum: Grundlagen der Programmierung": "PGP",
   "Einführung in die Rechnerarchitektur": "ERA",
+  "Grundlagenpraktikum: Rechnerarchitektur": "GRA",
   "Einführung in die Softwaretechnik": "EIST",
-  "Algorithmen und Datenstrukturen": "AD",
+  "Grundlagen: Algorithmen und Datenstrukturen": "GAD",
```

Comment:
This change Breaks the other Algorithm courses:
![image](https://user-images.githubusercontent.com/26258709/163852399-13527f95-7936-4c84-a8e7-fa926c548b54.png)

```suggestion
  "Grundlagen: Algorithmen und Datenstrukturen": "GAD",
  "Effiziente Algorithmen und Datenstrukturen": "EAD",
```

Reply:
Great catch, fixed with [052f8d4](https://github.com/TUM-Dev/CalendarProxy/pull/98/commits/052f8d40ab0af8291ce13583e86c1c31f2c0776f)